utils/hare-bootstrap

```diff
@@ -199,6 +199,19 @@ is_localhost() {
     [[ -e $path && $(cat $path) == $node ]] && return 0 || return 1
 }
 
+check_consul_failure() {
+
+    while read node bind_ip; do
+        if ssh -n $node "sudo systemctl --quiet --state=failed is-failed hare-consul-agent"; then
+            echo 'Failed to start agent on '$node: >&2
+            return 0 
```

Comment:
@SwapnilGaonkar7, why are we returning `0` if consul fails to start? shouldn't we be returning 1 and 0 in case of success?

Reply:
`why are we returning 0 if consul fails to start`
@mssawant  This function checks if consul is in failed state that's why we are returning 0(success) if consul fails to start

Reply:
@SwapnilGaonkar7 , ideally such functions return boolean, I see that you want to return True or False here. So why not do that instead? return True or False instead of 0 or 1, the code will be more readable, no?

Reply:
@mssawant  We can not write 'return true' in bash. Return only accept numeric value
Nearest match is 'true; return'
I have modified code accordingly

Reply:
Yeah, makes sense.